components/menus/planning.ts

```diff
@@ -286,6 +288,114 @@ export async function planningView(
         })
     }
 
+    let loadoutSlots = [
+        {
+            SlotName: "carriedweapon",
+            SlotId: "0",
+            Recommended: null,
+        },
+        {
+            SlotName: "carrieditem",
+            SlotId: "1",
+            Recommended: null,
+        },
+        {
+            SlotName: "concealedweapon",
+            SlotId: "2",
+            Recommended: {
+                item:
+                    contractData.Peacock?.noCarriedWeapon === true
+                        ? null
+                        : typedInv.find(
+                              (item) => item.Unlockable.Id === pistol,
+                          ),
+                type: "concealedweapon",
+            },
+        },
+        {
+            SlotName: "disguise",
+            SlotId: "3",
+            Recommended: {
+                item: typedInv.find((item) => item.Unlockable.Id === suit),
+                type: "disguise",
+            },
+        },
+        {
+            SlotName: "gear",
+            SlotId: "4",
+            Recommended: {
+                item:
+                    contractData.Peacock?.noGear === true
+                        ? null
+                        : typedInv.find((item) => item.Unlockable.Id === tool1),
+                type: "gear",
+            },
+        },
+        {
+            SlotName: "gear",
+            SlotId: "5",
+            Recommended: {
+                item:
+                    contractData.Peacock?.noGear === true
+                        ? null
+                        : typedInv.find((item) => item.Unlockable.Id === tool2),
+                type: "gear",
+            },
+        },
+        {
+            SlotName: "stashpoint",
+            SlotId: "6",
+            Recommended: null,
+        },
+        briefcaseId && {
+            SlotName: briefcaseProp,
+            SlotId: briefcaseId,
+            Recommended: {
+                item: {
+                    ...i,
+                    Properties: {},
+                },
+                type: i.Unlockable.Id,
+                owned: true,
+            },
+            IsContainer: true,
+        },
+    ].filter(Boolean)
+
+    /**
+     * Handles loadout lock for Miami and Hokkaido
+     */
+    const parentLocationProperties =
+        locationData.parents[sublocation?.Properties?.ParentLocation]
+            ?.Properties
+
+    if (parentLocationProperties?.LimitedLoadout) {
```

Comment:
```suggestion
    if (sublocation?.Properties?.LimitedLoadout) {
```
Loadouts should be available for A Silver Tongue and Patient Zero even before unlocking loadout slots in Miami and Hokkaido, respectively. The properties of the sublocation should be used here.

Reply:
ah!